modules/sagemaker/custom-kernel/modulestack.yaml

```diff
@@ -0,0 +1,44 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - "ecr:Describe*"
+              - "ecr:Get*"
+              - "ecr:List*"
+              - "sagemaker:CreateImage"
+              - "sagemaker:CreateImageVersion"
+              - "sagemaker:DescribeImage"
+              - "sagemaker:DescribeImageVersion"
+              - "sagemaker:DescribeAppImageConfig"
+              - "sagemaker:CreateAppImageConfig"
+              - "sagemaker:UpdateAppImageConfig"
```

Comment:
Module faild b/c it cannot add tags (sagemaker:AddTags):



```
----------------------------------- Create Image -----------------------------------
--
709 | Attempting to create Image: echo-kernel
710 | Warning: already exists [error: An error occurred (AccessDeniedException) when calling the CreateImage operation: User: arn:aws:sts::616260033377:assumed-role/addf-mlops-sagemaker-kernels-kernel-custom-kernel-074ff5b4/AWSCodeBuild-c7c3cf23-86b7-4283-8c5f-feacb9a239a0 is not authorized to perform: sagemaker:AddTags on resource: arn:aws:sagemaker:us-east-1:616260033377:image/echo-kernel because no identity-based policy allows the sagemaker:AddTags action]. Skipping creation...
711 | ----------------------------------- Create Image Version -----------------------------------
712 | Attempting to create image version with BaseImage: 616260033377.dkr.ecr.us-east-1.amazonaws.com/custom-kernel:latest
713 | No versions exist [error: An error occurred (ResourceNotFound) when calling the DescribeImageVersion operation: Image with ARN arn:aws:sagemaker:us-east-1:616260033377:image/echo-kernel does not exist]. Need to create
714 | Last Image Version: None
715 | Traceback (most recent call last):
716 | File "scripts/create_kernel.py", line 212, in <module>
717 | create_image_version_if_not_exists()
718 | File "scripts/create_kernel.py", line 92, in create_image_version_if_not_exists
719 | image_version = create_image_version()
720 | File "scripts/create_kernel.py", line 67, in create_image_version
721 | ImageName=image_name,
722 | File "/root/.venv/lib/python3.7/site-packages/botocore/client.py", line 530, in _api_call
723 | return self._make_api_call(operation_name, kwargs)
724 | File "/root/.venv/lib/python3.7/site-packages/botocore/client.py", line 960, in _make_api_call
725 | raise error_class(parsed_response, operation_name)
726 | botocore.errorfactory.ResourceNotFound: An error occurred (ResourceNotFound) when calling the CreateImageVersion operation: Image with ARN arn:aws:sagemaker:us-east-1:616260033377:image/echo-kernel does not exist

```

Reply:
Confirmed successful deployment once added